### PR TITLE
2.3.3 release notes

### PR DESCRIPTION
**Pull request by danielmewes**
@segphault

### CODE REVIEW

File: NOTES.md [hunk 1]
```diff
@@ -1,3 +1,64 @@
...
+
+No migration is required when upgrading from RethinkDB 2.3.x. Please read the
+[RethinkDB 2.3.0 release notes][release-notes-2.3.0] if you're upgrading from an
+older version.
+
+[release-notes-2.3.0]: https://github.com/rethinkdb/rethinkdb/releases/tag/v2.3.0
+
+### Windows support ###
+
+RethinkDB has been available for Windows since version 2.3.0. With RethinkDB 2.3.3, we
```
Comment (segphault) [inline NOTES.md]: The current phrasing is fine, but here's some alternate language to consider:

RethinkDB 2.3.0 was the first version to include native Windows compatibility. In RethinkDB 2.3.3, the Windows port is ready to emerge from "beta" testing. We now officially support RethinkDB on the Windows platform alongside our existing support for Linux and Mac OS X. We're also extending our [commercial support][comm-support] services to include RethinkDB on Windows.

Although RethinkDB is now stable on Windows, there are still a few remaining limitations that we are actively working to address. We also haven't yet carried out as much performance tuning on the Windows port as we have on the Linux and OS X releases.

File: NOTES.md [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+* JavaScript driver
+ * The `Connection` class is now exported from the RethinkDB JavaScript module (#5758)
+* Java driver
+ * Added the `clientPort` and `clientAddress` methods to the `Connection` class in the
+   Java driver (#5571)
+
+## Contributors ##
+
+Many thanks to external contributors from the RethinkDB community for helping
+us ship RethinkDB 2.3.3. In no particular order:
```
Comment (segphault) [inline NOTES.md]: Maybe remove the "In no particular order" since there is only one. :-)

File: NOTES.md (hunk 1 above)
Comment (danielmewes) [inline NOTES.md]: Awesome! That's much nicer.

File: NOTES.md (hunk 2 above)
Comment (danielmewes) [inline NOTES.md]: Haha good catch. I just copied it over from the previous release and didn't think about it.

**Pull request merged**